### ISSUE TITLE
Cannot add above £999 in a single transaction

### ISSUE THREAD

**Issue by dcb0303** [moneymanagerex/moneymanagerex#3699]
MMEX version:
 - [x] 1.5.x
 - [ ] 1.6.x

Note: bug reporters are expected to have verified the bug still exists
either in the last stable version of MMEX or on updated development code
(master branch).

Operating System:
 - [x] Windows
 - [ ] Mac OSX
 - [ ] Linux 

# Description of the bug

Please describe your issue with details.
Add screenshot or other files if needed.

# Reproduction

Is the bug reproducible? 
 - [x] Always 
 - [ ] Randomly 
 - [ ] Happened only once

Reproduction steps:

1. 
2. 
3. 

…

Expected result:

Actual result:

# Additional information

If you have a backtrace for a crash or a warning, paste it here.

**Comment by dcb0303** [comment]
If I want to pay, transfer or receive more than £999 I cannot do it as a single transaction. e.g. transfer monies to clear a £4000 credit card bill cannot be done in a single transaction

**Comment by whalley** [comment]
Duplicate of https://github.com/moneymanagerex/moneymanagerex/issues/3626

**Comment by dcb0303** [comment]
I probably did it error.  Couldn't navigate round the site or forum. Latest version of mmex, 1 cannot enter more than £999. I regularly enter thousands, when I settle my credit cards. ChrisSent from my Samsung Galaxy S10e - Powered by Three
-------- Original message --------From: Mark Whalley ***@***.***> Date: 13/07/2021  6:43 pm  (GMT+00:00) To: moneymanagerex/moneymanagerex ***@***.***> Cc: dcb0303 ***@***.***>, State change ***@***.***> Subject: Re: [moneymanagerex/moneymanagerex] Cannot add above £999 in a single transaction (#3699) 
Duplicate of #3626

**Comment by whalley** [comment]
Try the latest beta build here: https://ci.appveyor.com/project/moneymanagerex/moneymanagerex/build/job/trrjekob0qgp3s1x/artifacts

**Comment by tactilis** [comment]
@dcb0303

This is a known bug that is fixed for the upcoming 1.5.4 release.

As @whalley said, it is a duplicate of #3626 (and several other reports of this issue)
